browser/brave_ads/android/java/org/chromium/chrome/browser/dialogs/BraveAdsNotificationDialog.java

```diff
@@ -70,6 +70,7 @@ public boolean onTouch(View v, MotionEvent event) {
                     updateWindowPosition((int) event.getRawX() + mXDown);
                     break;
                 case MotionEvent.ACTION_UP:
+                    v.performClick();
```

Comment:
Why do we need to perform click here?

Reply:
There is a warning about `onTouch should call View#performClick when a click is detected [ClickableViewAccessibility]`.